### ISSUE TITLE
Replacing notifications does not make them time out

### ISSUE THREAD

**Issue by He-Ro** [dunst-project/dunst#419]
Run this command:
```bash
dunstify original -p -u l | xargs dunstify replacement -u l -r; dunstify reference -u l
```
Now two notifications are shown, one with the title "replacement" and one with the name "reference", which is a reference for the timeout time.
I have a timeout set for low priority notifications.

*What I see:*
The reference notification gets dismissed after the timeout has passed.
The replacement notification does not get dismissed.

*What was expected:*
The replacement notification gets dismissed just like the reference notification.
### Installation info

- Version: `Dunst - A customizable and lightweight notification-daemon v1.2.0-97-g3691b7e`
- Install type: [AUR Package](https://aur.archlinux.org/packages/dunst-git/)
- Distro and version: `Arch Linux`

**Comment by bebehei** [comment]
Fixed in #411 (see the last commit there)

It is because the recent replace flickerless PR and timeout in msecs got merged in parallel

**Comment by bebehei** [comment]
@He-Ro Could you please rebuild `dunst-git` package and test it, if it's fixed? Thanks in advance.

**Comment by He-Ro** [comment]
It seems to be working. The two notifications get closed at the same time with the current version of dunst `v1.2.0-137-g76857b6`. Cheers
